src/driver/Driver.ts

```diff
@@ -132,7 +132,7 @@ export interface Driver {
     /**
      * Prepares given value to a value to be persisted, based on its column type.
      */
-    prepareHydratedValue(value: any, column: ColumnMetadata): any;
+    prepareHydratedValue(value: any, column: ColumnMetadata | string): any;
```

Comment:
This is a pretty difficult change as we'd need to update all drivers to support this.

You can't create a `ColimnMetadata` to pass info this method?

Reply:
I think not because is the type of the value as a raw string... For this I propose create a separate function specific to parse the values in the `addSelectAndMap` method. I agree with is a difficult change that need to update all drivers for this I think is a good idea create other method.